third_party/blink/renderer/bindings/core/v8/record_replay_events.cc

```diff
@@ -85,10 +85,12 @@ UserEventProbe::UserEventProbe(const char* name,
     : name_(name ? String(name) : atomic_name),
       event_target_(event_target),
       is_callback_(is_callback) {
-  // Disabled by default, see https://linear.app/replay/issue/RUN-1251
   if (IsRecordingOrReplaying() &&
-      !FeatureEnabled("disable-collect-events") &&
-      !AreEventsDisallowed()) {
+      !AreEventsDisallowed() &&
+      // // Disabled by default (RUN-1251)
+      // !FeatureEnabled("disable-collect-events") &&
```

Comment:
rm commented out code

Reply:
You read it too fast. It was already fixed 10 seconds later 👀